bindings/python/core_types/generated/py_enumeration.cpp

```diff
@@ -0,0 +1,57 @@
+//------------------------------------------------------------------------------
```

Comment:
Python wrapper for enum should inherit from python IntNum class and therefore it would behave as true python enum. But this is out of the scope of this PR and should be done as a separate task, probably by Yaroslav.